Source/JavaScriptCore/runtime/DeferredWorkTimer.cpp

```diff
@@ -235,10 +230,10 @@ bool DeferredWorkTimer::cancelPendingWork(Ticket ticket)
 void DeferredWorkTimer::cancelPendingWorkSafe(JSGlobalObject* globalObject)
 {
     Locker locker { m_taskLock };
-    for (auto ticket : globalObject->m_objectsForTicket->keys()) {
-        if (!ticket->isCancelled())
-            cancelPendingWork(ticket);
-        m_tasks.append(std::make_tuple(ticket, [](DeferredWorkTimer::Ticket) mutable { }));
+    for (auto& ticket : *globalObject->m_tickets) {
+        if (!ticket.isCancelled())
```

Comment:
You should keep ticket as Ref<TicketData> in the iteration body. This is weak ptr and can be nulled

Reply:
Fixed